python/test/t_SquaredNormalFactory_std.py

```
@@ -0,0 +1,60 @@
+
+ot.TESTPREAMBLE()
+ot.RandomGenerator.SetSeed(0)
+
+factory = otexp.SquaredNormalFactory()
+
+# Negative tests: rejection paths for buildMethodOfMoments
+try:
+    badSample = ot.Sample([[x] for x in [1.0, 2.0, -1.0, 4.0]])
+    factory.build(badSample)
+    print("FAIL: expected exception for negative data")
+except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)